doc/man/8/rbd-nbd.rst

```diff
@@ -18,9 +18,9 @@ Synopsis
 Description
 ===========
 
-**rbd-nbd** is a client for RADOS block device (rbd) images like rbd kernel module.
-It will map a rbd image to a nbd (Network Block Device) device, allowing access it
```

Comment:
s/a rbd/an rbd/
s/a nbd/an nbd/

I know that this breaks the well-known rule that the indefinite article "a" comes before words beginning with consonants, but that rule isn't actually correct. The real rule is "a" comes before words that begin with a consonant sound. Both initialisms "rbd" and "nbd" begin with vowel sounds, so "an" goes here. I know this is weird and counter-intuitive, but it's precisely these little peculiarities that make me think that maybe robots won't take my job just yet.